fk/secretsend_test.go

```diff
@@ -21,10 +21,34 @@ func TestEncryptSecret(t *testing.T) {
 		t.Fatalf("error loading private key: %s", err)
 	}
 
-	var keyRing openpgp.EntityList = []*openpgp.Entity{&pgpKey.Entity}
+	t.Run("with an empty filename", func(t *testing.T) {
+		armoredEncryptedSecret, err := encryptSecret(secret, "", pgpKey)
+		assert.ErrorIsNil(t, err)
+
+		messageDetails := decryptMessageDetails(armoredEncryptedSecret, pgpKey, t)
+		assertMessageBodyMatchesSecretContent(messageDetails.UnverifiedBody, secret, t)
+		assert.Equal(t, "_CONSOLE", messageDetails.LiteralData.FileName)
+		if messageDetails.LiteralData.ForEyesOnly() != true {
+			t.Fatalf("expected secret to be For Eyes Only, but isn't")
+		}
+	})
+
+	t.Run("with a filename", func(t *testing.T) {
```

Comment:
what happens for an empty *file* ?

Reply:
(or one with only whitespace)

Reply:
![image](https://user-images.githubusercontent.com/254290/52121861-f13eb500-2618-11e9-81ec-c11df1edcebb.png)

Reply:
I'm gonna add this as a new bug so we can get this merged.

Reply:
Thanks! I've picked this up, but can drop commit if you'd rather!